tests/Browser/BookRoomsTest.php

```diff
@@ -143,6 +144,50 @@ public function testCannotBookUnavailableTime()
         });
     }
 
+     /**
+     * Create a blackout, and then attempts to book a room during that time
+     * assert that the page doesn't change to the booking creation. 
+     */
+    public function testCannotBookDuringBlackout()
+    {
+        $this->browse(function (Browser $browser){
+
+            $admin = User::first();
+            
+
+            if($admin === null) {
+
+                $admin = User::factory()->create();
+                $admin->assignRole('super-admin');
+            }
+
+            $browser->loginAs($admin);
+            $browser->visit('/bookings/search');
+
+            $browser
+                ->assertSourceHas('<title>CSU Booking Platform</title>');
+
+            $room = Room::inRandomOrder()->first();
+            $blackout=Blackout::factory()->create();
+            $blackout->rooms()->attach($room);
+
+            $browser->press("@room-select-".$room->id);
+
+            $browser->mouseover('#add-recurences-button');
+
+            $browser->within( new DateTimePicker('start_time_0'), function($browser) {
+                $browser->setDatetime(10,02);
+            })->pause(250);
+
+            $browser->within( new DateTimePicker('end_time_0'), function($browser) {
+                $browser->setDatetime(10,03);
+            })->pause(250);
+
+            $browser->pressAndWaitFor('#createBookingRequest');
+
+            $browser->assertSee('Create a booking request');
```

Comment:
You aren't asserting for the specific error message. You can stay on this page for another reason and this test would prove anything

Reply:
For some reason my commit didn't include the changes made after we noticed the first time. Updating now.